.circleci/config.yml

```diff
@@ -105,16 +105,16 @@ jobs:
 
   pypy3:
     docker:
-      - image: pypy:3.6-7.2.0
+      - image: pypy:3.6-7.3.1
```

Comment:
PyPy released 7.3.2, is there a image for it yet?

Reply:
I have downloaded 7.3.2 from the [download page](https://www.pypy.org/download.html) and the segfaults are still there.